Project.toml

```diff
@@ -1,15 +1,10 @@
 name = "AutoHashEquals"
 uuid = "97026771-8bcd-44fb-b7eb-f644370d63d3"
-authors = ["andrew cooke <andrew@acooke.org>"]
-version = "1.0.0"
+authors = ["Neal Gafter <neal@gafter.com>", "andrew cooke <andrew@acooke.org>"]
+version = "2.0.0"
```

Comment:
is this truly a breaking change? it seems backward compatible at least at my cursory docs-comparing level

Reply:
It depends on what we do with type arguments.

Reply:
Yes, it is now a breaking change.  `@auto_hash_equals` used to ignore type arguments.  Now they are significant by default.

Reply:
it sounds like this changed back, so it's no longer a breaking change